### PR TITLE
Update DCOS README for executing integration tests

### PR DESCRIPTION
**Pull request by orsenthil**
* Update README on how to run integration-tests.
* Remove the reference to non-existing file.

### CODE REVIEW

**Comment by orsenthil** [comment]
@mesosphere-mergebot label ready for review

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@karlkfi  - yes, we don't. 
screen shot 2017-11-08 at 8 15 39 pm

**Comment by karlkfi** [comment]
What’s the difference between py.test and pytest? Instructions say one but your screenshot says another.

**Comment by orsenthil** [comment]
@karlkfi , @mellenburg  - please review this PR.

File: README.md [hunk 1]
```diff
@@ -118,23 +118,20 @@ The tests can be run via Pytest while SSH'd as root into a master node of the cl
...
+    
+    This test user has a known login token with far future expiration. DO NOT USE IN PRODUCTION.
+    After the test, remember to delete the test user.
+    
+    For more information, see [User Management](https://dcos.io/docs/latest/administration/user-management/).
 
-    The `test_env.export` script tries to look up cluster metadata, but can't distinguish between public and private nodes yet. So those have to be manually specified.
 
-1. Run the tests with Pytest
+2. Run the tests using py.test in the cluster.
```
Comment (karlkfi) [inline README.md]: lets make this say `pytest` per https://stackoverflow.com/a/41893170/18852

**Comment by karlkfi** [comment]
For reference, per convo with @orsenthil:
The agents and private agents are now retrieved from Mesos. So specifying them via env var is optional. Since https://github.com/dcos/dcos/pull/1508

**Comment by margaret** [comment]
Note that you do have to declare the nodes in an enterprise cluster.

Should you mention `dcos-shell`? At least on CoreOS it seems like you still need to prepend that.

**Comment by orsenthil** [comment]
`teamcity/dcos/test/aws/cloudformation/simple ` unrelated to this change. Captured this in: https://jira.mesosphere.com/browse/DCOS_OSS-1841

**Comment by orsenthil** [comment]
@margaret  - this is open dcos/dcos README, since we had setup the environment in the earlier step, running `pytest` was fine. Showed the example test-execution in the comments too.

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Pull request merged**